### PR TITLE
Created contributing.md

### PR DESCRIPTION
**Pull request by ananya-agrawal**

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
Looks good. Please update description of PR to refer to issue for which this PR was made. Also it would look better if points can be of bullet style, I am not sure, would like to wait for other review. But update description.

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# How To Contribute
+### The following below mentioned is a set of guidlines for contributing to any project under [OpenCode](https://github.com/opencode18)
+#### These guidlines must be taken into consideration before creating any pull request.
+#### Comment Before Solving Issue
+It should be taken care that you first comment on the issue you wanted to work on and get it assigned to you. Assignment will be done on first come first serve basis.
+#### Pull request must not rejected if the rules are violated. 
+As this is to create an easy and joyful onboarding process for new programmers and first-time contributors.
+#### Single Commit per Pull request
```
Comment (gauthamzz) [inline contributing.md]: Remove this

**Comment by gauthamzz** [comment]
Add  if you face any doubts contact on zulip .

**Comment by mubaris** [comment]
Use Bullet Points

**Comment by gauthamzz** [comment]
Add add comments to the code.

**Comment by gauthamzz** [comment]
Link your issue to the commit. Or vice versa . When sending pr

**Comment by gauthamzz** [comment]
use bullet points via - tag not ####

**Comment by pradeepgangwar** [comment]
Also link issue you are working to in description. This time i am doing this for you. #91

**Pull request merged**